### DISCUSSION TITLE
When typing, CMD + Backspace or Shift + Backspace can delete selected element

### DISCUSSION THREAD

**Discussion by LefanTan** [bcakmakoglu/vue-flow#1383] [General]
Hi. I noticed that a Node is selected, and I'm current typing on an Input/Textarea, sometimes entering "Shift+Backspace" may delete the currently selected node, even though the Input/Textarea element is in focus.

Here's a GIF demo, where I'm typing random stuff and clicking `Shift + Backspace`, which will unexpectedly delete the currently selected node.
This feels like an unintended bug?
![move](https://github.com/bcakmakoglu/vue-flow/assets/12474777/77f0c2a7-2a3a-49d7-bfaf-d2a347808be0)

You can try it here
Demo: https://stackblitz.com/edit/vitejs-vite-pffl1o?file=src%2FApp.vue

**Comment by bcakmakoglu** [comment]
Looks like a bug, will be fixed in the next patch release :)

**Reply by bcakmakoglu** [reply]
@LefanTan Fixed with `1.33.6`
